### ISSUE TITLE
Sarumans Minor Decisions Overhaul

### ISSUE THREAD

**Issue by JukeJukestar** [MattTheLegoman/RealmsInExile#285]
<details>
<summary>Sarumans 5 Minor Decisions</summary>

![image](https://user-images.githubusercontent.com/76008882/152354065-a45ba82e-fcc1-4cec-badf-14b4b237ffe1.png)

</details>
Sarumans Minor Decisions regarding Uruk-Hai armies are imbalanced and partially dont really work.
Ill try to do a full review of Saruman when I have the time maybe next week but I cant guarantuee anything. This issue is regarding the Uruk-Hai path, I have not tested the Dunlending alternative yet.
I am not sure how much of this is dependent on monthly income and such, at the point of testing these I am Duke of Isengard and have a few extra holdings in Dunland. (Year is 3016)

####################

<details>
<summary>"Rally Uruk-Hai" Minor Decision</summary>

<img width="592" alt="image" src="https://user-images.githubusercontent.com/76008882/152342413-4afc8699-f86c-41aa-b28e-f55e15d0353b.png">

</details>
What it says it does:
Gives you 1.8k Men, you gain a modifier that buffs heavy infantry a bit for the duration of a year. Can only be done every 10 years. Costs 990 Gold and 500 Piety

<details>
<summary>Follow-Up event</summary>

<img width="1392" alt="hord2" src="https://user-images.githubusercontent.com/76008882/152342526-ee605168-f3e2-4b78-9ce3-8b194e45a324.png">

</details>
What it does:
You gain the modifier for one year, after one year a "Horde Ready" event fires that claims you get 1.8k Orks.

<details>
<summary>Event Troop Stack</summary>

![image](https://user-images.githubusercontent.com/76008882/152342892-82cb9ed6-d490-415c-ae91-b05b626d1aaf.png)

</details>

However, the actual event troops you get are far less. They are also completely composed of levy.
The exact number it claims to give you and the number it gives you constantly fluctuate.

Issues:
- It does not actually give you the ammount of troops it claims to give you
- The troops is completely composed of levy
- Paying 950 gold for 1000 levy is the worst trade deal ever, even if it gave 2000 levies this would never actually be useful, unless you maxed out your MAA.
- The modifier that increases Heavy Infantry strength is misleading (since you dont actually get heavy infantryat the end) and also pretty much useless for anything exept tracking the time it takes for the troops to train
- Theres really no reason for this decision to be used since MAA are more reliable, reinforce and are far more cost effective, making them superior in every situation even if the Army was larger and had MAA.
In my personal opinion it should just be removed to encourage Saruman to use his income to build up MAA and Isengards holdings instead.

###############################

<details>
<summary>"Train Uruk-Hai Crossbowmen" Minor Decision</summary>

![image](https://user-images.githubusercontent.com/76008882/152343698-f26b21b3-8ca2-4c30-bbaf-a0dc9ad88856.png)

</details>

What it says it does:
gives you 200 men, gives +10% archer damage for 1 year. can only be done every 10 years, costs 250 gold and piety

<details>
<summary>Actual Effects</summary>

![image](https://user-images.githubusercontent.com/76008882/152343897-ca7f4112-9813-4520-8d88-a9122f7b6e76.png)

</details>

What it does:
You gain the modifier for one year. after one year you receive the "Crossbowmen ready" event which gives 200 crossbowmen.
This works as intended

Issues:
- Just like the other decision, this is incredibly weak, with the difference that we have an exact comparison here. Hiring 200 uruk-hai crossbowmen MAA costs 180 gold and no piety, with the difference being that the event ones don't cost upkeep and do not reinforce at all.
- There really isnt any point to this decisions existance as you could just use the MAA system instead. The only possibly advantage would be being able to go over the MAA cap, but by the time saruman gets there 200 crossbowmen really wont matter.
In my opinion this decision should be removed too since Saruman should use all his income on building up MAA and Isengard itself.

########################

<details><summary>"Demand Tribute" Minor Decision</summary>

![image](https://user-images.githubusercontent.com/76008882/152347708-0d21faa8-af21-4374-9c54-e5ed69853137.png)

</details>

What says it does:
"Gain a county claim of your choice on a neighbouring realm and use a regiment of uruks" (bit vague description)

What it does:
Fires an event where you gain 10% levy size decrease for one year in exchange for a claim on a neighbouring county of your choice.

Issues:
- There is no reason to use this decision as you get county and duchy conquest casus belli from the start. Against the riddermark its also useless since your invasion of them is scripted
- 10% less levy for a year has nearly no impact, since the duration is so short it wont matter unless you are at war with roharn and scrambling for every soldier
- This decision should not exist in its current form. Vanilla systems can handle this instead. Saruman starts out with incredibly learning and diplomacy skills and often times has the "accomplished forger" (claim fabrication speed +75%) and "sanctioned loopholes" (buy claims on counties, duchies and rarely kingdoms/ empires for some piety) skills unlocked.
I am assuming this is a placeholder and its eventually supposed to give you tributaries, in this case I am unsure how you plan to handle it since its WIP.

###########################################

<details><summary>"Hire Orc Horde" and "Hire Uruk Warband" minor decisions</summary>

![image](https://user-images.githubusercontent.com/76008882/152349315-866792b2-9abf-45b8-9718-bf11d9a92165.png)

![image](https://user-images.githubusercontent.com/76008882/152349357-1c77c441-1a9e-45fa-8091-0322e71d8e1a.png)

</details>

What it does: Both of these decisions cost a decent chunk of gold (500 / 250) and piety (500 for both) to use and can only be used when at war. You instantly get an event troop stack that uses supply. (600 and 110 men respectively)
Troops are comprised of weaker Orc MAA and in the case of the second decision 10 siege ballistae.

<details><summary>Each decision also unlocks a respective innovation: </summary>

![image](https://user-images.githubusercontent.com/76008882/152354492-dcf996b2-e2a3-466e-b02e-a33a6027c0de.png)
Orc Pits - unlocks the weak but cheap mordor MAA

![image](https://user-images.githubusercontent.com/76008882/152354542-6e20d425-42f7-4983-8f5f-c89f7eada2a3.png)
Uruk-Hai Siegeworks - unlocks mordor Siege Engines

</details>

Issues:
- These decisions can be used repeatedly after a 5 year cooldown, giving you the innovation again which doesnt break the game but feels unellegant
- You can only use these decisions while at war, meaning you have to declare a war to get these innovations
- The cost effectiveness is bad, MAA are superior so you would use both of these decisions just for the innovations.

####################################

Possible Solutions:

The "Demand Tribute" Decision appears to be WIP so despite it being completely useless its fine.

Having 4 distinct minor decisions just to hire different amounts of event troop orcs feels overwhelming, and makes it hard to compare usefulness since their effects are quite vague. Because of this deciding between them is difficult as well.
The system could be streamlined by splitting up/ changing the decisions.

1. Instead of "Hire Orc Horde" and "Hire Uruk Warband" you gain two major decisions that can only be used once. "Recruit Misty Mountain Orcs" and "Recruit Uruk Engineers"
For gold and piety, after one year they give decent stack of event troops that cost upkeep and the respective special innovation.
(Weaker mordor MAA for the first decision;Some uruk MAA and some Uruk siege engines for the second decision)

This way, getting the innovations a one time investment with some reinforcing event troops that aren't instantaneous and can't be repeated. Less useful during wartime since it takes time to complete and the reward is more useful long term.

2. "Rally Uruk-Hai" merges with "Train Uruk-Hai Crossbowmen" and replaces "Hire Orc Horde / Hire Uruk Warband" as the War-time instant recruitment decisions.
While at war, you can use this decision to instantly get a decent chunk of non-reinforcing event troops without upkeep (comprised of levy, some uruk MAA and uruk crossbowmen) in exchange for a decent gold cost. This can be used once every couple years (either 5 or 10)

This is supposed to be the repeatable decision for when you need to quickly raise some troops during a war that isn't going too well

**Comment by JukeJukestar** [comment]
TLDR;
Instead of 4 repeatable orc recruitment decisions
(2 Minor Decisions that can only be used while at war, give instant troops that cost gold and an innovation)
(2 Minor Decisions that can be used anytime, take a year to complete and give non-reinforcing troops)

you have three decisions,
(2 Major decisions that take a year to complete, give troops that cost gold and an innovation)
(1 Minor decision that can only be used while at war, gives instant troops that are free but don't reinforce)

**Comment by Remerod** [comment]
JUke when having more then 2 pictures please use [The Spoiler feature](https://github.com/MattTheLegoman/RealmsInExile/wiki/Text-Tricks-on-GitHub) so it gets easier to **navigate** the issue,

**Comment by JukeJukestar** [comment]
@Remerod whoops, fixed now

**Comment by Remerod** [comment]
@JukeJukestar be sure to check the previews before you post, some pics now don't show because you forgot to do a new row

**Comment by jj248** [comment]
Good commentary here.  It's on my list to revisit Rohan and Saruman as part of v3.0 so moving to that project for triage.

**Comment by jj248** [comment]
These are on my near-term list of considerations @JukeJukestar so thanks again for bringing my attention to this again.  I'll close this now as it's linked to #536, but let's have a discussion about it in a forum post in suggestions. <3
